public/js/main-template.js

```diff
@@ -12,93 +12,122 @@ $(document).ready(function () {
     asideBarRight = document.getElementById("AsideBarRight");
     fieldContainer = document.getElementById("field-container");
     installcontainer = document.getElementById("install-container");
+   if (document.getElementById("pannelone")) {
+        document.getElementById("pannelone").classList.toggle("Active");
+        document.getElementById("AsideBarRight").classList.toggle("RightPanel1Active");
+        document.getElementById("RightPanelone").classList.toggle("Active");
 
-    // $(".LeftPannelIcon1").click(function () {
-    //     toggleInstallMenu();
-    // });
-
-    // $(".PannelIcon2").click(function () {
-    //     toggleRightInstallMenu();
-    // });
-  
-    
+    }
 });
 
-function PannelIcon2(event)  {
-    $(".AsideBarInstallRight").toggleClass("RightPanel2Active");
-    $(".RightPanel2").toggleClass("Active");
-    event.currentTarget.classList.toggle("Active")
-};
-
-function toggleRightInstallMenu() {
-    $(this).toggleClass("Active", { duration: 3000 });
-    $(".PannelIcon").removeClass("Active", { duration: 3000 });
-    $(".AsideBarInstallRight").toggleClass("RightPanel2Active");
-    $(".AsideBarRight").removeClass("RightPanel1Active");
-    $(".RightPanel1").removeClass("Active");
-    $(".RightPanel2").toggleClass("Active");
-
-    if ($(".AsideBarRight").hasClass("Active")) {
-        toggelProjectSelectDiv();
-        removeClasses([rightMenuBtn], "display-block");
-        addClasses([rightMenuBtn], "display-none");
+function toggelProjectSelectDiv(event, page) {
+
+    if (event != "") {
+        if (event.currentTarget.classList.contains("PannelIcon")) {

```

Comment:
event.currentTarget is repeating so better take it on top in one variable so you don't have to write again. this is DRY priniciple.

```suggestion
const currentTarget=event.currentTarget;
        if (currentTarget.classList.contains("PannelIcon")) {

```